react-native-gutenberg-bridge/index.js

```diff
@@ -89,4 +89,12 @@ export function requestImageUploadCancel( mediaId ) {
 	return RNReactNativeGutenbergBridge.requestImageUploadCancel( mediaId );
 }
 
+export function getOtherMediaOptions( filter, callback ) {
+	return RNReactNativeGutenbergBridge.getOtherMediaOptions( filter, callback );
+}
+
+export function requesOtherMediaPickFrom( mediaSource, multiple, callback ) {
```

Comment:
Minor typo here I believe: request is missing the T?

Also, just noting that I think you could shorten these a bit to just be: 

```
export const getOtherMediaOptions = RNReactNativeGutenbergBridge.getOtherMediaOptions;
export const requestOtherMediaPickFrom = RNReactNativeGutenbergBridge.requesOtherMediaPickFrom;
``` 

I personally prefer avoiding explicitly passing the arguments, but I don't think it's wrong the way you have it either.